TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -597,12 +597,11 @@
 				}
 
 				.neos-button-small {
-					height: 32px;
-					width: 32px;
-					padding: 0 10px;
+					height: $wideMargin;
+					width: $wideMargin;
```

Comment:
yeah exactly, if you looks at the css you can find a lot of usages of those constants everywhere and also calculations using them

yeah it would in this change, but actually used it in #532